types/pouchdb-core/index.d.ts

```diff
@@ -55,6 +55,15 @@ declare namespace PouchDB {
             rev: RevisionId;
             status: Availability;
         }
+        interface RevisionDiffList {
+            [documentId : string]: string[];
+        }
+        interface RevisionDiff {
+            [change : string]: string[];
```

Comment:
Another option would be this:

```typescript
interface RevisionDiff {
    missing?: string[];
    possible_ancestors?: string[];
    [reason: string]: string[];
}```

@spaulg Whatever you want to go with works for me.